### DISCUSSION TITLE
Télédéclarer pour l'utilisateur

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1997] [Idées]
**Situation :**
Dans le cadre de la campagne de TD actuelle, nous avons convenu que les biz dev aideraient exceptionnellement les utilisateurs à TD par souci d'efficacité. Or à ce jour, pour qu'un BD puisse télédéclarer en masse les diagnostics d'un utilisateur dont les cantines sont déjà existantes et qui ont déjà l'utilisateur comme gestionnaire, il faut qu'il TD un par un les diagnostics en créant un magic token et en cliquant sur "Valider et télédéclarer".

**Besoin :**
Pouvoir TD en masse les diagnostics d'un utilisateur en une seule manip.

**Possible solution :**
Ouvrir tous les droits aux BD pour les imports de fichiers, ce qui nous permettrait d'importer des fichiers avec la valeur "teledeclared" dans la dernière colonne sans pour autant que nous soyons gestionnaire de cette cantine.

@JenniferStephan @alemangui @hfroot, votre avis?

**Comment by hfroot** [comment]
Je voudrais bien voir comment on pourrait soutenir nos utilisateurs pour qu'ils soient contents d'utiliser la plateforme sans notre intervention. Une autre solution j'imagine pour ce pb c'est d'ajouter une fonctionnalité qui permets les utilisateurs de 5+ cantines (le nb c'est un exemple) de télédéclarer leur chiffres dans un seul coup eux-mêmes. Je créerai un design plus tard

**Comment by gauthierandre** [comment]
Effectivement, c'est ce dont on parlait hier @hfroot et c'est aussi notre objectif, d'à terme, de travailler sur l'autonomisation. 

Dans la perspective d'un échange plus complet sur les différentes pistes d'amélioration  pour l'utilisation de ma cantine par nos gestionnaires — notamment dans un contexte d'obligation de remontée de données —, je vais finaliser un petit tableau qui revient sur plusieurs problèmes. Les solutions existantes et en quoi il faut les améliorer, avec des idées des gestionnaires pour que leur navigation soit plus fluide. 

Ça serait cool de prendre un temps d'échange, en présentiel, pour aborder tous ces sujets ; ce sera le moment de préparer la prochaine campagne.
